test/framework/flux.go

```diff
@@ -915,12 +1076,16 @@ func (e *ClusterE2ETest) gitBranch() string {
 	return e.ClusterConfig.FluxConfig.Spec.Branch
 }
 
-func (e *ClusterE2ETest) clusterConfGitPath() string {
+func (e *ClusterE2ETest) clusterConfGitPathFromName(clusterName string) string {
```

Comment:
nit: I know we use `Conf` in that other method but can we spell it out in these new methods?
```suggestion
func (e *ClusterE2ETest) clusterConfigGitPathFromName(clusterName string) string {
```